api/proto/teleport/legacy/types/events/events.proto

```diff
@@ -3693,6 +3695,81 @@ message ElasticsearchRequest {
   string Query = 12 [(gogoproto.jsontag) = "query"];
 }
 
+// DynamoDBRequest is emitted when a user executes a DynamoDB request.
+// We support DynamoDB access via app access and database access,
+// therefore this event includes either DatabaseMetadata or AppMetadata.
+message DynamoDBRequest {
+  // OperationPlane specifies DynamoDB request operation plane.
+  enum OperationPlane {
+    // GENERAL is for otherwise uncategorized operations.
+    GENERAL = 0;
+    // CONTROL is for control-plane operations: create and manage DynamoDB tables.
+    CONTROL = 1;
+    // DATA is for data-plane operations: CRUD operations on data in a table.
+    DATA = 2;
+    // STREAM is for stream operations: enable/disable a stream on a table or access data modification records in a stream.
+    STREAM = 3;
+    // TRANSACTION is for transaction opertions that provide ACID guarantees.
+    TRANSACTION = 4;
+  }
+
+  // Metadata is a common event metadata.
+  Metadata Metadata = 1 [
+    (gogoproto.nullable) = false,
+    (gogoproto.embed) = true,
+    (gogoproto.jsontag) = ""
+  ];
+
+  // User is a common user event metadata.
+  UserMetadata User = 2 [
+    (gogoproto.nullable) = false,
+    (gogoproto.embed) = true,
+    (gogoproto.jsontag) = ""
+  ];
+
+  // SessionChunkID is the ID of the session that was created for this 5 minute
+  // application log chunk.
+  string SessionChunkID = 3 [(gogoproto.jsontag) = "session_chunk_id"];
+
+  // AWS contains extra AWS metadata of the request.
+  AWSRequestMetadata AWS = 4 [
```

Comment:
AWSRequestMetadata contains the servicename and the region. In case of DynamoDB access the service is constant.

Reply:
hmm but region and host are not constant...